dashboard/package.json

```diff
@@ -10,24 +10,6 @@
     "@clr/ui": "^5.4.1",
     "@improbable-eng/grpc-web": "^0.14.0",
     "@paciolan/remote-component": "^2.11.0",
-    "@types/diff": "^5.0.1",
-    "@types/google-protobuf": "^3.15.3",
-    "@types/js-yaml": "^4.0.2",
-    "@types/json-schema": "^7.0.8",
-    "@types/jsonwebtoken": "^8.5.4",
-    "@types/lodash": "^4.14.171",
-    "@types/moxios": "^0.4.12",
-    "@types/pako": "^1.0.2",
-    "@types/qs": "^6.9.7",
-    "@types/react": "^17.0.15",
-    "@types/react-copy-to-clipboard": "^5.0.1",
-    "@types/react-jsonschema-form": "^1.7.6",
-    "@types/react-router-dom": "^5.1.8",
-    "@types/react-router-hash-link": "^2.4.1",
-    "@types/react-tabs": "^2.3.3",
-    "@types/react-transition-group": "^4.4.2",
-    "@types/semver": "^7.3.8",
-    "@types/ws": "^7.4.7",
```

Comment:
Moved to dev deps. Also, I removed all of them and just installed those ones Typescript was complaining (this way we reduce the types dependencies (as many of them are including also the definition of the types)